app/controllers/shopify_app/authenticated_controller.rb

```diff
@@ -3,6 +3,12 @@ module ShopifyApp
   class AuthenticatedController < ActionController::Base
     include ShopifyApp::Authenticated
 
-    protect_from_forgery with: :exception
+    protect_from_forgery with: :exception, unless: :jwt_shopify_domain?
+
+    private
+
+    def jwt_shopify_domain?
```

Comment:
Could we name this something that describes what it's telling us? like `session_token_auth?` or `valid_session_token_header?` or something?